Upstream: sento to mailing list
http://lists.gnupg.org/pipermail/gnupg-devel/2015-April/029729.html

From e0c55062d02b4cb0f4972f067eb84c824853defe Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Sat, 18 Apr 2015 10:52:09 +0200
Subject: [PATCH] configure.ac: Use PKG_PROG_PKG_CONFIG

..instead of hard-coding pkg-config with AC_PATH_PROG.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 646bc32..cf5c592 100644
--- a/configure.ac
+++ b/configure.ac
@@ -342,7 +342,7 @@
 
 dnl check for pkg-config
 if test "$pinentry_gtk_2" != "no" -o "$pinentry_gnome_3" != "no"; then
-        AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
+        PKG_PROG_PKG_CONFIG
 	if test x"${PKG_CONFIG}" = xno ; then
 		pinentry_gtk_2=no
 		pinentry_gnome_3=no
@@ -416,7 +416,7 @@
 
 dnl check for pkg-config
 if test "$libsecret" != "no"; then
-        AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
+        PKG_PROG_PKG_CONFIG
 	if test x"${PKG_CONFIG}" = xno ; then
 		libsecret=no
 	fi
-- 
2.3.3

